PrabhaK/03 - JavaScript/project/Florist/order.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Order</title>
+      <link rel="stylesheet" href="order.css">
+   </head>
+   <body>
+      
+    
+    <div class="whole-data">
```

Comment:
Give proper name.
instead of whole-data use page-container.

<h3>APPLY THIS TO ALL PLACES.</h3>